dfastmi/batch/AnalyserAndReporterDflowfm.py

```diff
@@ -266,13 +269,13 @@ def analyse_and_report_dflowfm(
                     t = tide_bc[i]
                     key = (q,t)
                 else:
-                    t = 0
+                    t = 'N/A'
```

Comment:
make it None instead

Reply:
```suggestion
                    t = None
```

Reply:
Might be better if t is renamed into tide_boundary

Reply:
Discussed with @ralph-peelen-deltares , will be tackled in the refactor of this module.